packages/js/components/src/plugins/index.tsx

```diff
@@ -37,7 +37,7 @@ export const Plugins = ( {
 	onAbort,
 	onComplete,
 	onError = () => null,
-	pluginSlugs = [ 'jetpack', 'woocommerce-services' ],
+	pluginSlugs = [ 'woocommerce-services' ],
```

Comment:
This is a default for all instances of `<Plugins>`. I've updated the code further down so each task using `<Plugins>` (only shipping & tax anyway) explicitly specifies the plugin slugs.

I've kept this optional not to disrupt too much but removed `'jetpack'` as WCS&T won't require Jetpack anymore.